MAUI/Themes/Keys.md

```diff
@@ -204,7 +204,7 @@ This page lists the keys for each control and the element to which it is mapped
             <br/>
         </td> 
         <td>
-            Color of the Autocomplete background in normal state.
+            Color of the SfAutocomplete background in normal state.
```

Comment:
Remove this changes and currently share the link alone in the PR

Reply:
Changes addressed sir.